inception-project/inception
# #1103 Metric Selection for Recommender Evaluation

**What's in the PR**
* The Evaluation Metric to be used to plot learning curve can be changed from the drop down on the side panel. 

**How to test manually**
* Set up a recommender for your project
* Annotate
* Open the recommender tab
* it should now be possible to change the displayed metric via a dropdown above the y-axis

**Automatic testing**

**Documentation**
